docs/styleguide/generic.rst

```diff
@@ -162,3 +162,52 @@ Comments
       do_something()
 
 * Avoid end of line comments.
+
+Git
+===
+
+Feature branch development
```

Comment:
Maybe just remove `Git` and make ` Feature branch development` as the top level heading. with ` Don't revert, just push the fix ` as subheading / sub-section.

Even if we would switch to mercurial or go back to Bazaar, the same principle should apply.